kebbie/cmd.py

```diff
@@ -148,3 +175,33 @@ def cli():
         for c in correctors:
             c.emulator.show_keyboards()
             print(f"Predictions : {c.emulator.get_predictions()}")
+
+    elif args.cmd == "get_page_source":
+        correctors = instantiate_correctors(args.keyboard, get_layout=False)
+
+        for c in correctors:
+            # Get the page source
+            page_source = ET.fromstring(c.emulator.driver.page_source)
+
+            # Get the keyboard package name
+            keyboard_package = emulator.KEYBOARD_PACKAGE.get(args.keyboard, None)
+
+            if keyboard_package:
+                # Filter elements that have the specified package
+                filtered_elements = [element for element in page_source if element.get("package") == keyboard_package]
+
+                if filtered_elements:
+                    # If there are filtered elements, create a new XML with those elements
+                    filtered_page_source = ET.Element(page_source.tag, page_source.attrib)
+                    filtered_page_source.extend(filtered_elements)
+                    page_source = filtered_page_source
+
+            page_source_str = ET.tostring(page_source, encoding="utf8").decode("utf8")
+
+            # Print the keyboard elements to the console if specified
+            if args.print_page_source:
+                print(page_source_str)
+
+            # Save the keyboard elements to a file
+            with open(args.keyboard + "_" + args.page_source_file, "w", encoding="utf-8") as file:
```

Comment:
Users should be able to save the file in a different folder :
```suggestion
            with open(args.page_source_file, "w", encoding="utf-8") as file:
```